.gitignore

```diff
@@ -2,3 +2,4 @@
 *.d
 bin/
 lib/
+build/
```

Comment:
cmake prefers to have not in-source builds. would be great to keep it if you don't mind

Reply:
feel free to reject..